examples/provider/bid/pulsepoint/pulsepoint.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>PulsePoint - PubFood Example page</title>
+  </head>
+  <body>
+    <script src="../../../../build/pubfood.js"></script>
+    <script>
+        var PulsepointPubFoodAdapter=function(e){function n(e,n,o,i){var r;for(r in n){var a=n[r],c=a.getParam(l);c.placementCode=a.name,c.elementId=a.elementId,t(c,o,i,e)}}function t(e,n,i,r,a){if(window.pp&&window.pp.Ad){var c=function(t){o(e,t,n,i,r)},d=new window.pp.Ad({cf:e.cf,cp:e.cp,ct:e.ct,cn:1,ca:window.pp.requestActions.BID,cu:s,adUnitId:e.placementCode,callback:c});d.display()}else a&&m*a>p&&o(e,null,n,i,r),setTimeout(function(){t(e,n,i,r,(a||0)+1)},50)}function o(e,n,t,o,r){n?(c[r][e.placementCode]={html:n.html,params:e},t({slot:e.placementCode,value:n.bidCpm,targeting:{hb_pp_bidder:l,hb_pp_bid:n.bidCpm.toFixed(2),hb_pp_adid:e.placementCode,hb_pp_sessionid:r}})):c[r][e.placementCode]=null,i(r)&&o()}function i(e){var n;for(n in d[e]){var t=d[e][n];if(!c[e].hasOwnProperty(t.name))return!1}return!0}function r(e,n,t){try{var o=c[t][n].html,i=c[t][n].params;if(o){for(var r=i.cf.toUpperCase().split("X"),d=r[1],l=r[0],p=e.getElementsByTagName("IFRAME"),m=0;m<p.length;m++)p[m].parentNode.removeChild(p[m]);var u=document.createElement("iframe");u.width=l,u.height=d,u.style.border="0px",u.scrolling="no",u.frameborder=0,u.allowtransparency=!0,e.body.appendChild(u),u.contentWindow.contents=o,u.src='javascript:window["contents"]',e.defaultView&&e.defaultView.frameElement&&(e.defaultView.frameElement.width=l,e.defaultView.frameElement.height=d);var s=u.contentWindow?u.contentWindow.document:u.contentDocument.document?u.contentDocument.document:u.contentDocument;s.body&&(s.body.style.margin="0")}}catch(f){a(f)}}function a(e){console&&(console.error?console.error(e):console.log&&console.log(e))}e=e||{};var c={},d={},l="pulsepoint",p=e.libTimeout||1e3,m=e.libCheckTimeout||50,u=e.libUri||window.location.protocol+"//ads.contextweb.com/getjs.static.js",s=e.bidUri||window.location.protocol+"//bid.contextweb.com/header/tag";window.ppRender=window.ppRender||function(e,n,t){r(e,n,t)},this.bidProvider=function(){var e=function(e,t,o){var i=Math.floor(1e7*Math.random());d[i]=e,c[i]={},n(i,e,t,o)};return{name:l,libUri:u,init:e,refresh:e}}};
```

Comment:
Typically we'd like to see the BidProvider delegate code un-minified and such that it illustrates the use of the underlying provider API.

Could you `addBidProvider()` with un-minified code so that Pubfood users can understand and reason about the code.